### ISSUE TITLE
[BUG] - loops in loopProperty

### ISSUE THREAD

**Issue by MrVauxs** [fantasycalendar/FoundryVTT-Sequencer#146]
**Describe the bug**
Defining `loops` in loopProperty's options causes the effect to not loop at all, even if the `loops` is the same as presumed default `0`.

**To Reproduce**
Copy and run the following macro
```js
new Sequence()
    .effect()
        .from(token)
        .duration(2000)
        .scaleToObject(token.document.texture.scaleX)
        .loopProperty("spriteContainer", "scale.x", { from: 1, to: 2, delay: 800, duration: 300, pingPong: true })
        .loopProperty("spriteContainer", "scale.y", { from: 1, to: 2, delay: 800, duration: 300, pingPong: true, loops: 0 })
    .play();
```
**Expected behavior**
In the example above, both axis should be changing sizes. 

**Screenshots**

**Setup:**
 - Foundry Version: 10.291
 - System Version: PF2e 4.8.3
 - Sequencer Version: 2.412
 - Browser & version: Google Chrome Version 110.0.5481.178

**Active modules:**
 - Sequencer
 - socketlib

**Comment by MrVauxs** [comment]
Checking the code, there is either a typo on the wiki saying that the default is `loops: 0,` or there is a typo in the code with 
https://github.com/fantasycalendar/FoundryVTT-Sequencer/blob/ebe2a977f6407c0b063379d003955d8ed8fc3206/scripts/sections/traits/animation.js#L125

Still, no animations work with any other `loops` defined

**Comment by Haxxer** [comment]
Fixed in 2.413
